cpp/modules/deck.gl/core/src/lib/layer.cpp

```diff
@@ -366,6 +359,12 @@ void Layer::invalidateAttribute(const std::string& name, const std::string& diff
 //   }
 // }
 
+void Layer::_drawState(wgpu::RenderPassEncoder pass) {
```

Comment:
There are four "layer lifecycle" functions that all layer subclasses need to define:
- initializeState
- updateState
- drawState
- finalizeState

In JavaScript we decided against having underscore on these, as they are talked about in documentation etc.